relay-server/src/services/spooler/mod.rs

```diff
@@ -1128,6 +1149,10 @@ impl BufferService {
 
             BufferState::Disk(ref disk) => health.0.send(!disk.is_full().await.unwrap_or_default()),
         }
+        relay_statsd::metric!(
+            timer(RelayTimers::SpoolerHealthDuration) = check_start.elapsed(),
```

Comment:
Do we need this? We already have `handle_health` which tells us how long the operation takes and we have a good indication that this is very very fast. Also for project configs relays (where we have the issue) this is always Memory.